code/modules/mob/living/living_defense.dm

```diff
@@ -268,6 +268,10 @@
 		return FALSE
 	if(user.pull_force < move_force)
 		return FALSE
+	// This if-statement checks if the user is horizontal, and if the user either has no martial art, or has judo, drunk fighting or krav, in which case it should also fail
+	if(IS_HORIZONTAL(user) && (!user.mind.martial_art || istype(user.mind.martial_art, /datum/martial_art/judo) || istype(user.mind.martial_art, /datum/martial_art/drunk_brawling) || istype(user.mind.martial_art, /datum/martial_art/krav_maga)))
```

Comment:
Could use use `is_type_in_list()` here instead?

Reply:
Didn't know that existed, I'll go test around!